fala/apps/adviser/tests/test_search_page_journeys_playwright.py

```diff
@@ -22,9 +14,25 @@ def test_postcode_search_journey(self):
         ]
         for postcode in test_cases:
             with self.subTest(postcode=postcode):
-                self.visit_search_page()
-                self.page.get_by_label("Postcode").fill(postcode)
-                self.page.get_by_role("button", name="Search").click()
-                expect(self.page.locator("h1")).to_have_text("Search results")
-                expect(self.page.get_by_role("listitem").filter(has_text=f"Postcode: {postcode}")).to_be_visible()
-                expect(self.page.get_by_text("in order of closeness")).to_be_visible()
+                page = self.visit_results_page(postcode)
+                expect(page.h1).to_have_text("Search results")
+                expect(page.listitem_for(postcode)).to_be_visible()
```

Comment:
this is doing the same thing as `page.change_search_grey_box.nth(1)).to_have_text("Organisation: Islington Law Centre"` (but with postcode). It would be nice to consolidate those.

Reply:
Have made the test clearer to understand.